Formula/gator.rb

```diff
@@ -0,0 +1,20 @@
+class Gator < Formula
+  desc "CLI Utility for Open Policy Agent Gatekeeper"
+  homepage "https://open-policy-agent.github.io/gatekeeper/website/docs/gator"
+  url "https://github.com/open-policy-agent/gatekeeper.git",
+    tag:      "v3.9.0",
+    revision: "dfd2dd80734e20d30c00f7e729f03ef5ee9730ed"
+  license "Apache-2.0"
+  head "https://github.com/open-policy-agent/gatekeeper.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "gator"
+    bin.install "bin/gator"
```

Comment:
Can we use
```ruby
system "go", "build", *std_go_args
```
?

Reply:
Yeah I can switch to that. Went this way for the built in ldflags with the make target, but I can also do it with go build